tests/runner.py

```diff
@@ -69,7 +69,9 @@ def main(test_path, test_pattern, coverage_filepath):
     start_time = time.time()
     for mod_tests in suite:
         if mod_tests.countTestCases():
-            runner = xmlrunner.XMLTestRunner(output=output_file, verbosity=2)
+            runner = xmlrunner.XMLTestRunner(stream=sys.stdout,
```

Comment:
Is this all you had to do so that all test output would be in the same stream- or do we still need changes e.g. to `print()` in test classes?

Reply:
Yes.  I did more digging and found that `XMLTestRunner` extends `TextTestRunner`.  `TextTestRunner` sets a `stream` parameter to `stderr` if it isn't defined.  So I defined it and told it to send to `stdout` instead.  Irritated at myself that it took me that long to find this.

Reply:
You kidding? This is great! :+1: